.github/workflows/automerge.js

```diff
@@ -0,0 +1,137 @@
+let branch = '';
+let actor = '';
+let prName = '';
+
+if (process.argv.length >= 3) { branch = process.argv[2]; }
+if (process.argv.length >= 4) { prName = process.argv[3]; }
+if (process.argv.length >= 5) { actor = process.argv[4]; }
+
+console.log('--------------------------------------------------------');
+
+if (!branch)
+{
+    console.log('Branch name argument (first) was not specified');
+    return;
+}
+
+if (!prName)
+{
+    console.log('prName argument (second) was not specified');
+    return;
+}
+
+if (!actor)
+{
+    console.log('Actor argument (third) was not specified');
+    return;
+}
+
+console.log(branch);
+console.log(prName);
+console.log(actor);
+
+if (actor != 'dependabot')
+{
+    console.log('Automerge works only for PRs created by dependabot.');
+    return;
+}
+
+if (prName.includes('patternfly'))
+{
+    console.log('Automerge cant merge patternfly Prs.');
+    return;
+}
+
+if (prName.includes('types/node'))
+{
+    console.log('Checking for types node version.');
+    let pattern = /from 16[.]\d\d[.]\d\d to 16[.]\d\d[.]\d\d/;
```

Comment:
```suggestion
    let pattern = /from 16[.]\d+[.]\d+ to 16[.]\d+[.]\d+/;
```

(because `\d\d` wont' match 16.0.1 or 16.17.189)

Reply:
Oh, ok, I though that versiosns are strictly defined as 2 digits.

Reply:
Hm, I see, that was probably situation, when I had opened only dependabots PRs with two digits, today, I have many single digits :)